src/main/java/Events.java

```diff
@@ -0,0 +1,28 @@
+public class Events extends Task {
+    private final String EVENTS_SYMBOL = "[E]";
+    private String time;
+
+    public Events(String taskName, String time) {
+        super(taskName);
+        setTime(time);
+    }
+
+    public String printTaskDescription() {
+        return EVENTS_SYMBOL + super.printTaskDescription() + String.format("(at: %s)",getTime());
```

Comment:
Some coding standard, include a space after the comma

Reply:
Agreed. I would change the command to comply with coding standards.